src/iris/api.py

```diff
@@ -278,6 +278,7 @@ def ts_to_sql_datetime(ts):
     `plan_notification`.`step` as `step`,
     `plan_notification`.`repeat` as `repeat`,
     `plan_notification`.`wait` as `wait`,
+    IF(`plan_notification`.`optional` = 1, 'True', 'False') as `optional`,
```

Comment:
BOOLEAN also gets rid of this awkward bit

Reply:
Not sure if I can get rid of this since BOOLEAN is also stored as TINYINT(1).

Reply:
This selects the strings "True" and "False". Instead, we should just go with the raw value of 0 or 1 and have the caller cast it.

Reply:
Removed this from select statement and moved the translation from 0 -> "False" and 1 -> "True" to the Jinja template.